### PR TITLE
Add activity notes and test files

### PR DESCRIPTION
**Pull request by jgarciamesa**
Added brief notes about the activity on the README.
Created empty test files.

### CODE REVIEW

File: test_dir/file1.txt
Comment (LishaRamon) [inline test_dir/file1.txt:1]: Would be nice to have something to read here :)

Comment (certifiedp) [reply test_dir/file1.txt:1]: Naruto

File: README.md [hunk 1]
```diff
@@ -13,3 +13,8 @@ The dates so far have been:
 
 The INTERSECT project is supported by NSF awards [2017424](https://www.nsf.gov/awardsearch/showAward?AWD_ID=2017424)
 and [2017259](https://www.nsf.gov/awardsearch/showAward?AWD_ID=2017259).
+
+## Comments -- Acrivity
```
Comment (LishaRamon) [inline README.md]: Activity** but it's a comment so shouldn't really matter

File: test_dir/file1.txt
Comment (jgarciamesa) [reply test_dir/file1.txt:1]: Added text on commit [c787bbb](https://github.com/jgarciamesa/intersect-colab-git/pull/2/commits/c787bbbc49ec0c0758324506deebd5724e31361d).

File: README.md (hunk 1 above)
Comment (jgarciamesa) [reply README.md]: Thanks! Fixed on [3f0b907](https://github.com/jgarciamesa/intersect-colab-git/pull/2/commits/3f0b9077fc471c9ddbbe046f6527b7d4dd35057a).

File: test_dir/file1.txt
Comment (LishaRamon) [reply test_dir/file1.txt:1]: > Naruto

IMG_6345_20240714_231214

**Comment by LishaRamon** [comment]
LGTM!

**Review by certifiedp — APPROVED** [review]

**Pull request merged**